tests/src/unit/shared/formatConversion/formatConversionTestUtil.ts

```diff
@@ -1,199 +1,5 @@
-}
-
-export function partialBlocksToBlocksForTesting<
-  BSchema extends BlockSchema,
-  I extends InlineContentSchema,
-  S extends StyleSchema,
->(
-  schema: BlockNoteSchema<BSchema, I, S>,
-  partialBlocks: Array<PartialBlock<NoInfer<BSchema>, NoInfer<I>, NoInfer<S>>>,
-): Array<Block<BSchema, I, S>> {
-  return partialBlocks.map((partialBlock) =>
-    partialBlockToBlockForTesting(schema.blockSchema, partialBlock),
-  );
-}
-
-export function partialBlockToBlockForTesting<
-  BSchema extends BlockSchema,
-  I extends InlineContentSchema,
-  S extends StyleSchema,
->(
-  schema: BSchema,
-  partialBlock: PartialBlock<BSchema, I, S>,
-): Block<BSchema, I, S> {
-  const contentType: "inline" | "table" | "none" =
-    schema[partialBlock.type!].content;
-
-  const withDefaults: Block<BSchema, I, S> = {
-    id: "",
-    type: partialBlock.type!,
-    props: {} as any,
-    content:
-      contentType === "inline"
-        ? []
-        : contentType === "table"
-          ? {
-              type: "tableContent",
-              columnWidths: undefined,
-              headerRows: undefined,
-              headerCols: undefined,
-              rows: [],
-            }
-          : (undefined as any),
-    children: [] as any,
-    ...partialBlock,
-  };
-
-  Object.entries(schema[partialBlock.type!].propSchema).forEach(
-    ([propKey, propValue]) => {
-      if (
-        withDefaults.props[propKey] === undefined &&
-        propValue.default !== undefined
-      ) {
-        (withDefaults.props as any)[propKey] = propValue.default;
-      }
-    },
-  );
-
-  if (contentType === "inline") {
-    const content = withDefaults.content as InlineContent<I, S>[] | undefined;
-    withDefaults.content = partialContentToInlineContent(content) as any;
-  } else if (contentType === "table") {
-    const content = withDefaults.content as TableContent<I, S> | undefined;
-    withDefaults.content = {
-      type: "tableContent",
-      columnWidths:
-        content?.columnWidths ||
-        content?.rows[0]?.cells.map(() => undefined) ||
-        [],
-      headerRows: content?.headerRows || undefined,
-      headerCols: content?.headerCols || undefined,
-      rows:
-        content?.rows.map((row) => ({
-          cells: row.cells.map((cell) => partialContentToInlineContent(cell)),
-        })) || [],
-    } as any;
-  }
-
-  return {
-    ...withDefaults,
-    content: partialContentToInlineContent(withDefaults.content),
-    children: withDefaults.children.map((c) => {
-      return partialBlockToBlockForTesting(schema, c);
-    }),
-  } as any;
-}
-
-export function addIdsToBlock(block: PartialBlock<any, any, any>) {
-  if (!block.id) {
-    block.id = UniqueID.options.generateID();
-  }
-  if (block.children) {
-    addIdsToBlocks(block.children);
-  }
-}
-
-export function addIdsToBlocks(blocks: PartialBlock<any, any, any>[]) {
-  for (const block of blocks) {
-    addIdsToBlock(block);
-  }
-}
+export * from "@shared/formatConversionTestUtil.js";
```

Comment:
let's for now remove this file altogether and directly reference `@shared/formatConversionTestUtil`. This makes the codebase consistent and less "in limbo" state.

If we want to prioritize moving all unit tests, we should discuss this with 3 of us. Atm I'm not immediately convinced; afaik we decided on moving the formatting tests to a shared directory, because they shared similar logic - but we didn't settle on a rule to "have all unit tests" in `/tests`